templates/es-client.yaml

```diff
@@ -97,7 +97,17 @@ spec:
         volumeMounts:
         - name: storage
           mountPath: /data
+        {{- if .Values.configmap.enabled }}
+        - name: cfgmap
```

Comment:
The common pattern is to call this volume config

Reply:
Sorry, I don't understand what you mean, but feel free to edit the code